shell/platform/fuchsia/flutter/flatland_connection.cc

```diff
@@ -71,6 +80,42 @@ void FlatlandConnection::DoPresent() {
   fuchsia::ui::composition::PresentArgs present_args;
   present_args.set_requested_presentation_time(0);
   present_args.set_acquire_fences(std::move(acquire_fences_));
+
+  // Schedule acquire fence overflow signaling if there is one.
+  if (acquire_overflow_ != nullptr) {
+    FML_CHECK(acquire_overflow_->event_.is_valid());
+    async::PostTask(dispatcher_, [dispatcher = dispatcher_,
+                                  overflow = acquire_overflow_]() {
+      const size_t fences_size = overflow->fences_.size();
+      std::shared_ptr<size_t> fences_completed = std::make_shared<size_t>(0);
+      std::shared_ptr<std::vector<async::WaitOnce>> closures;
+
+      for (auto i = 0u; i < fences_size; i++) {
+        auto wait = std::make_unique<async::WaitOnce>(
+            overflow->fences_[i].release(), ZX_EVENT_SIGNALED, 0u);
```

Comment:
AFAIK this release() causes a leak, see tqr/778771.

Reply:
Done.